pkgs/development/libraries/qt-5/hooks/qtbase-setup-hook.sh

```diff
@@ -62,3 +62,59 @@ postPatchMkspecs() {
 if [ -z "$dontPatchMkspecs" ]; then
     postPhases="${postPhases}${postPhases:+ }postPatchMkspecs"
 fi
+
+_Qt_sortless_uniq() {
+	# `uniq`, but keeps initial order.
+	# This is to remove risks of combinatorial explosion of plugin paths.
+	cat -n | sort -uk2 | sort -nk1 | cut -f2-
```

Comment:
Looks to exist at https://stackoverflow.com/a/20639730 - Stackoverflow links are sometimes worthwhile comments